tests/flows/fulfillment/test_configuration.py

```diff
@@ -414,13 +414,15 @@ def test_fulfill_configuration_order(mocker):
 
     fulfill_configuration_order(mocked_client, mocked_order)
 
-    assert len(mocked_pipeline_ctor.mock_calls[0].args) == 5
+    assert len(mocked_pipeline_ctor.mock_calls[0].args) == 7
 
     assert isinstance(mocked_pipeline_ctor.mock_calls[0].args[0], Step)
```

Comment:
Could you fix here also

1. Compare with the list
2. Put concrete steps that you are using, not just basic class `Step`